src/components/restaurant/RestaurantCard.tsx

```diff
@@ -35,17 +35,25 @@ export default function RestaurantCard(restaurant: Restaurant): ReactElement {
         }}
       >
         <Header
-          cuisine={restaurant.cuisine}
-          distance={restaurant.distance}
-          priceLevel={restaurant.priceLevel}
+          // cuisine={
+          //   'Italian'
+          //   // todo: this should use a promotions cuisine, google place details doesn't have this
+          //   // restaurant.cuisine
+          // }
+          // distance={
+          //   1500
+          //   // // todo: where is this coming from?
+          //   // restaurant.distance
+          // }
```

Comment:
@andrewzulaybar I did not see where we can get this information from the place details. Do you mind explaining how we are getting these values?

Reply:
Oh yes cuisine should be from the promotion object and distance I think we still have to calculate on the FE

Just added this for distance:
https://promopal.atlassian.net/browse/PP-89

Reply:
Oh okay this might be difficult since many promotions can belong to the same restaurant, and I can't confidently say the cuisine will be the same for all promotions of the same restaurant. (for example fusion restaurant chinese + mexican, we have a chinese promo and a mexican promo for same restaurant)

Reply:
Ahh I see what you mean... Okay I think this complicates things lol I've created a ticket to think about it more

But I think we can probably implement some algorithm to show maybe the top 2/3 cuisine types for the restaurant based on the promotions or something like that

https://promopal.atlassian.net/browse/PP-91